debian/rules.in

```diff
@@ -9,7 +9,7 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CUDA_OPT = $(shell echo ${DEB_BUILD_PROFILES} | grep -qw cuda \
+CUDA_OPT = $(shell echo ${PATH} | grep -qw cuda \
```

Comment:
Maybe to check nvcc presence `command -v nvcc` and its version `nvcc --version`?

Reply:
@tzafrir-mellanox would this work for you?